indico/modules/events/sessions/controllers/management/sessions.py

```diff
@@ -123,3 +129,49 @@ def _process_PATCH(self):
             updates['colors'] = colors
         update_session(self.session, updates)
         return jsonify()
+
+
+class RHSessionPersonList(RHManageSessionsActionsBase):
+    """List of persons in the session"""
+
+    def _process(self):
+        session_ids = {s.id for s in self.sessions}
+        session_persons = (ContributionPersonLink
+                           .find(ContributionPersonLink.contribution.has(Contribution.session_id.in_(session_ids)))
+                           .all())
```

Comment:
`.distinct(ContributionPersonLink.person_id)` or you get the person once for each contribution he's associated with